L1TriggerScouting/OnlineProcessing/test/L1ScoutingSelector.py

```diff
@@ -158,12 +176,19 @@
     productLabel = cms.string("EtSum")
 )
 
+process.FinalBxSelectorBMTFStub = cms.EDProducer("MaskOrbitBxScoutingBMTFStub",
+    dataTag = cms.InputTag("l1ScBMTFUnpacker", "BMTFStub"),
+    selectBxs = cms.InputTag("FinalBxSelector", "SelBx"),
+    productLabel = cms.string("BMTFStub")
+)
+
 process.MaskedCollections = cms.Sequence(
     process.FinalBxSelectorMuon +
     process.FinalBxSelectorJet +
     process.FinalBxSelectorEGamma +
-    process.FinalBxSelectorTau +
```

Comment:
you dont need the Tau selector anymore?

Reply:
Ah, that was unintended from one of the last commits. Just for clarity, in the standard L1 Scouting operations, we remove the Tau collections from the selection stream as we have no use for that and to reduce the amount of storage required (taus would take almost half of the storage!). In the test script, we should leave it as it is meant for test purposes. I put it back